buildenv/aix/jdk11/yum_install_aix-ppc64.txt

```diff
@@ -18,33 +18,33 @@
 #
 # SPDX-License-Identifier: EPL-2.0 OR Apache-2.0 OR GPL-2.0 WITH Classpath-exception-2.0 OR LicenseRef-GPL-2.0 WITH Assembly-exception
 
-# This file installs package dependencies for building OpenJDK V10 with OpenJ9 on AIX 64-bit systems
+# This file installs package dependencies for building OpenJDK V11 with OpenJ9 on AIX 64-bit systems
```

Comment:
Not sure how this file got past the copyright check, the copyright end date should be 2019.

Reply:
Done

Reply:
The copyright end date should be 2019.

Reply:
done